core/src/test/scala/org/apache/spark/scheduler/SparkListenerSuite.scala

```diff
@@ -19,14 +19,17 @@ package org.apache.spark.scheduler
 
 import java.util.concurrent.Semaphore
 
-import scala.collection.mutable
```

Comment:
Sorry it was Idea's default imports reorganiser, I will apply the setting shown here:
https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Imports